packages/components/src/templates/badge.stories.ts

```diff
@@ -32,15 +32,21 @@ export const ButtonWithBadge = {
         </sd-button>
         <sd-button variant="secondary" size="md">
           <sd-icon name="system/phone"></sd-icon>
-          <sd-badge variant="error" size="md">4</sd-badge>
+          <sd-badge variant="error" size="md">
+            <span class="sr-only">Calls</span>
+            4
```

Comment:
I would call it out "5 Calls" instead of "5 Calls"?

Reply:
you mean instead of Calls 5?

Yes, I would prefer that as well.

Reply:
Updated